pytest_selenium/pytest_selenium.py

```diff
@@ -287,10 +291,6 @@ class DriverAction(argparse.Action):
 
     def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, values)
-        driver = getattr(drivers, values.lower())
-        # set the default host and port if specified in the driver module
-        setattr(namespace, 'host', getattr(driver, 'HOST', None))
-        setattr(namespace, 'port', getattr(driver, 'PORT', None))
```

Comment:
I think this means we can remove the custom action entirely?

Reply:
Probably, didn't think to try it.